<launch>

    <param name="use_sim_time" value="true"/>
    <node pkg="tf2_ros" name="static_tf" type="static_transform_publisher" args="0 0 0.4 0 0 0 1 base_link velodyne"/>
    <!-- <node pkg="lio_sam" type="lio_sam_imuPreintegration"   name="lio_sam_imuPreintegration"    output="screen" 	respawn="false"/> -->
    <node pkg="particle_filter_3d" name="test_particle_filter" type="test_particle_filter" output="screen">
        <rosparam command="load" file="$(find particle_filter_3d)/param/config_degen.yaml" />
    </node>
    <node pkg="particle_filter_3d" name="test_lidar_degen" type="test_lidar_degen" output="screen">
        <param name="max_range" value="10.0"/>
        <param name="degen_rate" value="0.1"/>
        <param name="lidar" value="velodyne_points"/>
    </node>
    <node pkg="particle_filter_3d" name="test_traj_record" type="test_traj_record" output="screen">
        <param name="folder" value="$(find particle_filter_3d)/trajectory/warehouse/"/>
        <param name="gt" value="xx.txt"/>
        <param name="trajectory" value="degen_10_denoise.txt"/>
    </node>

</launch>